Front-End/src/PageM.js

```diff
@@ -429,7 +378,6 @@ function AuxiliarySystemsDataTable(props) {
 function ForebarDataTable(props) {
     const [advice, setAdvice] = useState("");
     const [isLoaded, setIsLoaded] = useState(false);
-
```

Comment:
Some spaces can be helpful to leave in, if it makes the code easier to mentally separate into blocks. This is up to a personal preference, but sometimes empty lines can help break up code to make it easier to read.

If there are sections of code you want to break up, I'm all for leaving extra lines in there to make it easier to read. Otherwise, removing unneeded empty lines is always a plus